nixos/modules/services/network-filesystems/davfs2.nix

```
@@ -39,23 +41,30 @@ in
       '';
     };
 
-    extraConfig = mkOption {
-      type = types.lines;
-      default = "";
-      example = ''
-        kernel_fs coda
-        proxy foo.bar:8080
-        use_locks 0
-      '';
+    settings = mkOption {
+      type = types.submodule {
+        freeformType = format.type;
+      };
+      default = {};
+      example = literalExpression '' {
+        kernel_fs = "coda";
+        proxy = "foo.bar:8080";
+        use_locks = 0;
+      } '';
```

```suggestion
      example = literalExpression ''
        {
          kernel_fs = "coda";
          proxy = "foo.bar:8080";
          use_locks = 0;
        }
      '';
```
otherwise the doc renders it wrong